apps/ledger-live-mobile/src/screens/ClaimNft/ClaimNftQrScan.tsx

```diff
@@ -174,13 +174,13 @@ const ClaimNftQrScan = () => {
         ) : (
           <>
             <Flex
-              backgroundColor="neutral.c40"
+              backgroundColor="constant.black"
```

Comment:
If the camera component is unmounted, using `constant.black` as a background will make this unmounted state unnoticeable.